lib/pagseguro/items.rb

```diff
@@ -24,7 +30,9 @@ def <<(item)
     # Verify if the item is already included to item list.
     # Returns boolean.
     def include?(item)
-      @store.find {|stored_item| stored_item.id == ensure_type(Item, item).id }
+      @store.detect do |stored_item|
+        stored_item.id == item.id && stored_item.description == item.description
```

Comment:
@lucasrenan But if add a item with same `id` and `description` by now we will update the first one quantity.